src/chttpd/src/chttpd_auth_request.erl

```diff
@@ -34,7 +34,10 @@ authorize_request_int(#httpd{path_parts=[]}=Req) ->
 authorize_request_int(#httpd{path_parts=[<<"favicon.ico">>|_]}=Req) ->
     Req;
 authorize_request_int(#httpd{path_parts=[<<"_all_dbs">>|_]}=Req) ->
-    Req;
+   case config:get("chttpd", "admin_only_all_dbs", "false") of
+       "false" -> Req;
+       _Else -> require_admin(Req)
```

Comment:
I'd rather that "true" was required for this explicitly.

Reply:
Cool, done in latest push, added `Else -> couch_log:error` so folks know what’s up when this crashes, hope that was okay